static/src/javascripts/projects/common/modules/commercial/user-adblock-message.js

```diff
@@ -0,0 +1,14 @@
+define([
+    'common/utils/cookies'
+], function (
+    cookies
+) {
+    //right now we assume that the user who is a member (any of 3 paid tiers) and do not want to see ads, also won't see adblock messages
+    function isUserMember() {
+        return cookies.get('gu_adfree_user') ? true : false;
```

Comment:
I agree this is definitely an assumption. The defining characteristic is that the user has a paid tier, not necessarily that they get ad-free.